webfontkit-20190809-101721/generator_config.txt

```diff
@@ -0,0 +1,5 @@
+# Font Squirrel Font-face Generator Configuration File
+# Upload this file to the generator to recreate the settings
+# you used to create these fonts.
```

Comment:
do you know if this .txt file is needed to be included in the repo in order for it to work?